### DISCUSSION TITLE
Looking for a GOOD QUALITY 3" to 5" color LCD display

### DISCUSSION THREAD

**Discussion by Rollmops67** [Bodmer/TFT_eSPI#2846] [Compatible displays and setup files]
Hello from France.

I allready did do some projects with the fantastic TFT_eSPI library, with different color LCD screens coming mainly from Aliexpress.
All these screens are rather cheap, but the image quality (mostly the view angle and the color rendition) is rather poor.

So I'm looking for something with a better image quality (ISP ?) and a resolution between 480x320 and 800x600 pixels (about 4" in size).

If somebody knows such a product, I would be happy to know where to find it !

Roland

**Comment by Bodmer** [comment]
Have a look at this vendors products:
https://www.buydisplay.com/

**Reply by Rollmops67** [reply]
Thank you Bodmer for the link.
I'm intersted in a 5“ SSD1963 TFT Module but I'm a little confused with the interface to choose :
I don't know the difference between "6800 parallel interface" and "8080 parallel interface".
Can you please tell me what to choose ? (the display will be connected to an ESP32S3)

Thank you !

Roland

**Comment by Bodmer** [comment]
8080 is the one you want.

**Comment by Rollmops67** [comment]
This display from buydisplays. com just arrived :
https://www.buydisplay.com/lcd-5-inch-ips-ssd1963-tft-module-touch-screen-display-800x480-mcu-arduino

And there is no possible comparison with the cheap displays you find on Aliexpress !
Color rendering, contrast and view angles are clearly in another league.
A real eye candy, thank you for the tip.
I even managed to have smooth backlight changing (driven by a LDR), with the help of this topic :
https://github.com/Bodmer/TFT_eSPI/discussions/2194

Roland
